### PR TITLE
Add metrics for AWS API client calls.

### PR DESCRIPTION
**Pull request by dgoodwin**

### CODE REVIEW

File: pkg/awsclient/client.go [hunk 1]
```diff
@@ -46,6 +48,16 @@ const (
 	awsCredsSecretAccessKey = "aws_secret_access_key"
 )
 
+var (
+	metricAWSAPICalls = prometheus.NewCounterVec(
```
Comment (csrwng) [inline pkg/awsclient/client.go]: @dgoodwin should this be registered somewhere?

Comment (dgoodwin) [reply pkg/awsclient/client.go]: LOL yes thank you. My bad.

Comment (dgoodwin) [reply pkg/awsclient/client.go]: Updated.

File: pkg/controller/controlplanecerts/controlplanecerts_controller.go [hunk 2]
```diff
@@ -86,7 +86,7 @@ func NewReconciler(mgr manager.Manager) reconcile.Reconciler {
 // AddToManager adds a new Controller to mgr with r as the reconcile.Reconciler
 func AddToManager(mgr manager.Manager, r reconcile.Reconciler) error {
 	// Create a new controller
-	c, err := controller.New(controllerName, mgr, controller.Options{Reconciler: r})
+	c, err := controller.New("controlplanecerts-controller", mgr, controller.Options{Reconciler: r})
```
Comment (csrwng) [inline pkg/controller/controlplanecerts/controlplanecerts_controller.go]: thx!

**Pull request merged**